### DISCUSSION TITLE
How do I set up higher hp?

### DISCUSSION THREAD

**Discussion by HiddenStriker** [Pryaxis/TShock#3014] [Q&A]
I was messing with the config files of my server and I thought it could be fun to set up a raid boss player type of event. Though despite changing Max HP and Mana in config to 2-3000, it is impossible in game to go higher than 500 HP and maximum mana without buffs. Am I missing something? Is it just not possible?

**Comment by punchready** [comment]
That config is for preventing (or allowing) clientside cheats

**Reply by HiddenStriker** [reply]
Ah so I can't ever have a player above the max limits of the game itself?

**Comment by sgkoishi** [comment]
> Ah so I can't ever have a player above the max limits of the game itself?

The game will just do nothing when you use crystal/fruit after 400/500 HP. It is possible to use a plugin (or write your own) to listen for item usage and increase max HP correspondingly.

**Reply by HiddenStriker** [reply]
Sorry for the late reply, but I wish I could write plugins :/ I don't know the first thing about code. I'll be on the lookout if such a plugin is ever written that increases the max HP limit to a few thousands.

**Comment by zyrafaq** [comment]
i think its impossible unless u use server side player saves
